server/paymentProviders/paypal/adaptive.js

```diff
@@ -97,7 +97,9 @@ export default {
             paymentMethod.currency,
           )} across ${totalTransactions} transactions between ${firstTransactionAt} and ${lastTransactionAt}`,
         );
-        return 200000 + totalSpent; // total spent has a negative value (in cents)
+        // total spent has a negative value (in cents)
+        const initialBalance = paymentMethod.data.balance || 200000;
```

Comment:
What's `200000`?

Reply:
It's the max amount for PayPal pre-approval. I've just moved it to a `paypalPreApprovalLimit` variable to make things clearer.